### ISSUE TITLE
return value of cocopp.main

### ISSUE THREAD

**Issue by nikohansen** [numbbo/coco#1645]
is a dictionary which also contains the data from the background algorithms. The latter are indistinguishable from the data generated from the input arguments. This seems rather undesirable to me.

**Comment by brockho** [comment]
I agree. Not clear in which use case returning the background algorithms makes much sense.

**Comment by nikohansen** [comment]
The information necessary to distinguish fore- and background algorithm datasets is in `sortedAlgs` and `dictAlg` in `rungenericmany.main` when compared against `genericsettings.background` (or `.foreground_algorithm_list`).

**Comment by brockho** [comment]
To come back to this issue: What would we like to have as return value of `cocopp.main`? Only the data from the foreground algorithms? Or two return values, one for the foreground and one for the background algorithms?

Once we specify what we want, I can take care of this issue.

**Comment by nikohansen** [comment]
My first thought: return the data of the "foreground" algorithms and store both lists in the module as a `global` variable where it is convenient (which should already be the case?).
